hyp3lib/raster_boundary2shape.py

```diff
@@ -1,17 +1,15 @@
 """generates boundary shapefile from GeoTIFF file"""
 
-from __future__ import print_function, absolute_import, division, unicode_literals
-
-import os
-import sys
 import argparse
-from scipy import ndimage
+import os
+
 from osgeo import ogr
-from hyp3lib.asf_geometry import raster_meta, geotiff2boundary_mask, data_geometry2shape
+from scipy import ndimage
 
-# from hyp3lib.asf_time_series import raster_metadata
+from hyp3lib.asf_geometry import raster_meta, geotiff2boundary_mask, data_geometry2shape
 
 
+# from hyp3lib.asf_time_series import raster_metadata
```

Comment:
I'd lean towards dropping this comment, although it's outside the scope of the PR so doesn't have to happen.  I assume the intent was to show users how to import the `raster_metadata` function?

Reply:
There is a duplicate `raster_metadata` function that is similar, but not the same in `asf_time_series`:
https://github.com/asfadmin/hyp3-lib/blob/develop/hyp3lib/asf_time_series.py#L250
that had been imported at some point and then dropped for this duplicate one. 

I left the comment as a reminder to circle back and clean that up at some point, but it could just be an issue or a more expressive comment